components/Home/CommunityStats.tsx

```diff
@@ -0,0 +1,32 @@
+import { Card, Col, Container, Row } from 'react-bootstrap';
+
+export const CommunityStats = () => (
+  <div className="py-5 bg-white">
+    <Container className="text-center">
+      <h2 className="mb-4 text-dark">社区数据</h2>
+      <div className="d-flex justify-content-center mb-4">
+        <div
+          className="border-bottom border-warning"
+          style={{ width: '60px', borderBottomWidth: '3px !important' }}
+         />
+      </div>
```

Comment:
```suggestion
      <h2 className="mb-4 text-dark">社区数据</h2>
      <div className="d-flex justify-content-center mb-4">
        <div
          className="border-bottom-3 border-warning"
          style={{ width: '3.75rem' }}
         />
      </div>
```
1. https://getbootstrap.com/docs/5.3/utilities/borders/#width
2. 抽公共组件